modules/gltf/src/lib/extensions/KHR_draco_mesh_compression.ts

```diff
@@ -76,9 +81,24 @@ async function decompressPrimitive(
   const dracoOptions = {...options};
   // TODO - remove hack: The entire tileset might be included, too expensive to serialize
   delete dracoOptions['3d-tiles'];
-  const decodedData = await DracoLoader.parse(bufferCopy, dracoOptions, context);
+  const decodedData = await parse(bufferCopy, DracoLoader, dracoOptions, context);
```

Comment:
Especially having the type checks on the returned data would give me a lot of piece of mind. That was kind of the whole point of the big exercise I made.
```suggestion
  const decodedData = await parse(bufferCopy, DracoLoader, dracoOptions, context) as DracoMeshData;
```